### ISSUE TITLE
Pyautogui typewrite types numbers only

### ISSUE THREAD

**Issue by PaulBobyrev** [asweigart/pyautogui#70]
Hello!
My Pyautogui typewriter doesn't want to type letters. For example when I'm executing
`pyautogui.typewrite("abc123")`
only "123" appears. I have a Windows 7 machine, Python 3.5 and Pyautogui 0.9.33. In what the problem can consist in? Any suggestions will be greatly appreciated!

**Comment by Wu4m4n** [comment]
in what kind of input do you try to write that? in a input html or in a windows input?

**Comment by PaulBobyrev** [comment]
In Windows Python IDLE, Notepad, any kind of browsers and text editors. My final goal is to input text a website.

**Comment by asweigart** [comment]
Does the following code work?

`>>> pyautogui.press('1')
>>> pyautogui.keyDown('1')`

**Comment by PaulBobyrev** [comment]
Yes, both are working for numbers, but doesn't do it for letters

**Comment by asweigart** [comment]
Sorry, I misread. I meant, try pyautogui.press('a') and pyautogui.keyDown('a'). I want to know if the problem is with typewrite() or all the keyboard functions.

**Comment by PaulBobyrev** [comment]
Never mind. It seems that any keyboard functions related to letters won't work, but works perfectly with numbers. What is even stranger, this issue can be observed both on my laptop and PC, but the only thing that is the same is that both computers works under Russian edition of Windows 7.

**Comment by AlbericC** [comment]
>>> pyautogui.press('a')
    q

looks like the keyboard layout (azerty/fr here) affects `typewrite` and `press`

**Comment by setvisible** [comment]
Hello!

Here a workaround:

```python
import pyautogui
import pyperclip

def _workaround_write(text):
    """
    This is a work-around for the bug in pyautogui.write() with non-QWERTY keyboards
    It copies the text to clipboard and pastes it, instead of typing it.
    """
    pyperclip.copy(text)
    pyautogui.hotkey('ctrl', 'v')
    pyperclip.copy('')

# Test
text_with_special_chars = '@/:;\\.ABCabc?!~^[]{}()'
pyautogui.write(text_with_special_chars) 
# >>> //;\.ABCabc?§¨[]5°      -> NOK

_workaround_write(text_with_special_chars) 
# >>> @/:;\.ABCabc?!~^[]{}()       -> OK :)
```
It solved the issue on my Windows 64-bits machine / AZERTY french keyboard

**Comment by MoSchaub** [comment]
doesn't work on mac os
